DOCS-2494: Create custom ML training script, upload to registry, and submit training job (#3071)
viamrobotics/docs: docs/registry/_index.md

The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:

{{< cards >}}
{{% manualcard link="#modular-resources" %}}

**Modular resources** that add capabilities to your machine beyond what is built into `viam-server`

{{% /manualcard %}}
{{% manualcard link="#ml-models" %}}

**ML models** to deploy with machine applications like computer vision

{{% /manualcard %}}
{{% manualcard link="#ml-training-scripts" %}}

> **Training scripts** to train custom machine learning models from Python source distribution.

Author: ```suggestion
**Training scripts** to train machine learning models in the Viam cloud from custom Python scripts.
```

> **Training scripts** to train machine learning models in the Viam cloud from custom Python scripts.

Author: @tahiyasalam Is this accurate or would you know of a more descriptive way to phrase it?

Reviewer: Maybe `Python source distributions used to train and produce models in the Viam cloud for custom machine learning`?

Reviewer 2: Python source distributions sounds to me like the source code for python 3.9 o or other versions...

Reviewer: That's fair, we can Python scripts

> **Training scripts** to train machine learning models in the Viam cloud from custom Python scripts.

Reviewer 2: How about this? It avoids the awkwardness of using scripts twice
```suggestion
**Training scripts** that you can run in the Viam cloud to train machine learning models to your exact specifications.
```

> **Training scripts**, Python source distributions to train and produce models in the Viam cloud for custom machine learning

Author: ```suggestion
**Training scripts**, Python scripts to train and produce ML models in the Viam cloud for custom machine learning
```

See [Machine Learning](/services/ml/) for more information.

The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
You can also [upload your own model to the registry](/services/ml/upload-model/).

You can search the available ML models from the Viam registry here:

<div id="searchboxML"></div>
<p>
<div id="searchstatsML"></div></p>
<div class="mr-model" id="">
  <div class="modellistheader">
    <div class="name">Model</div>
    <div>Description</div>
  </div>
<div id="hitsML" class="modellist">
</div>
<div id="paginationML"></div>
</div>
<noscript>
    <div class="alert alert-caution" role="alert">
        <h4 class="alert-heading">Javascript</h4>
        <p>Please enable javascript to see and search ML models.</p>
    </div>
</noscript>

To use an existing model from the registry, [deploy the ML model to your robot](/services/ml/deploy/) and use a [Vision service](/services/vision/) to make detections or classifications on-machine.

## ML training scripts

> The Viam registry hosts trained ML training scripts, which you can use to train custom machine learning models from Python source distributions.

Author: ```suggestion
The Viam registry hosts custom Python ML training scripts, which you can use to train custom machine learning models.
```

You can also [upload your own model to the registry](/services/ml/upload-model/).

You can search the available ML models from the Viam registry here:

<div id="searchboxML"></div>
<p>
<div id="searchstatsML"></div></p>
<div class="mr-model" id="">
  <div class="modellistheader">
    <div class="name">Model</div>
    <div>Description</div>
  </div>
<div id="hitsML" class="modellist">
</div>
<div id="paginationML"></div>
</div>
<noscript>
    <div class="alert alert-caution" role="alert">
        <h4 class="alert-heading">Javascript</h4>
        <p>Please enable javascript to see and search ML models.</p>
    </div>
</noscript>

To use an existing model from the registry, [deploy the ML model to your robot](/services/ml/deploy/) and use a [Vision service](/services/vision/) to make detections or classifications on-machine.

## ML training scripts

The Viam registry hosts trained ML training scripts that users have made public, which you can use to train custom machine learning models from Python source distribution.
You can also upload your own training script.

> <!-- TODO: link to new machine learning custom training page -->

Reviewer 2: 👍

You can search the available ML training scripts from the Viam registry here:

<div id="searchboxScripts"></div>
<p>
<div id="searchstatsScripts"></div></p>
<div class="training-scripts" id="">
  <div class="modellistheader">
    <div class="name">Script</div>
    <div>Description</div>
  </div>
<div id="hitsScripts" class="modellist">
</div>

> <div id="paginationScripts"></div>

Reviewer 2: You need to change the corresponding css. Look for `.paginationML` and add `.paginationScripts` to the list for that css to be applied to.
Nice work adding this otherwise

Author: Done I think in `assets/scss/_styles_project.scss`
